chatkit-core/config/baseline.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" ?>
+<SmellBaseline>
+  <Blacklist></Blacklist>
+  <Whitelist>
+    <ID>LongMethod:RoomStore.kt$RoomStore$applyUserSubscriptionEvent</ID>
+    <ID>LongParameterList:CurrentUser.kt$CurrentUser$( id: String? = null, name: String, pushNotificationTitleOverride: String? = null, isPrivate: Boolean = false, customData: CustomData? = null, userIds: List&lt;String&gt; = emptyList(), callback: (Result&lt;Room, Error&gt;) -&gt; Unit )</ID>
```

Comment:
After the removal task it would be good to cleanup this list too, making it 90%+ shorter :)

Reply:
haha i expect it to be so 👍